tempesta_fw/cache.c

```diff
@@ -526,8 +527,10 @@ __cache_entry_size(TfwHttpResp *resp, TfwHttpReq *req)
 	/* Add status line length + CRLF */
 	size += resp->s_line.len + SLEN(S_CRLF);
 
-	/* Add body size */
+	/* Add body size accounting CRLF after the last chunk */
 	size += resp->body.len;
+	if (resp->flags & TFW_HTTP_CHUNKED)
+		size += SLEN(S_CRLF);
```

Comment:
Will the lengths and offset calculations break if we get '\n' only as EoL for chunks?